File: src/primus/pawel/Exercise_064.java
package primus.pawel;

import java.util.Arrays;

// Count of positives / sum of negatives
// 8kyu
// https://www.codewars.com/kata/576bb71bbbcf0951d5000044/train/java
// 2021-04-10 T:04:25:50
class Exercise_064 {

    public static int[] countPositivesSumNegatives(int[] input) {
        if (input.length == 0 || input == null) return new int[]{};

        int positivesCount = (int) Arrays.stream(input)
                .filter(n -> n > 0)
                .count();
        int negativesSum = (int) Arrays.stream(input)
                .filter(n -> n < 0)
                .sum();
        return new int[]{positivesCount, negativesSum};
    }


    public static void main(String[] args) {
        countPositivesSumNegatives(new int[]{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, -11, -12, -13, -14, -15});
    }
}

File: src/JS/Exercise_064.js
// Can we divide it?
// 8kyu
// https://www.codewars.com/kata/5a2b703dc5e2845c0900005a/train/javascript
// 2022-05-25 T:21:44:37
function isDivideBy(number, a, b) {
    return (number % a === 0 && number % b === 0)
}

console.log(isDivideBy(-12, 2,-6))